### PR TITLE
Fix i18n variable names and `)s` syntax for TE.json (Tegulu)

### PR DESCRIPTION
**Pull request by lukebarnard1**

### CODE REVIEW

File: src/i18n/strings/te.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "Alias (optional)": "అలియాస్ (ఇవచు ఇవకపపోవచు)",
     "all room members": "అన్ని గదుల సభ్యులు",
     "You do not have permission to post to this room": "మీకు ఈ గదికి పోస్ట్ చేయడానికి అనుమతి లేదు",
-    "You have been invited to join this room by %(inviterName)s": "% (InviterName) లు ఈ గదిలో చేరడానికి మీరు ఆహ్వానించబడ్డారు",
+    "You have been invited to join this room by %(InviterName)s": "%(InviterName)s ఈ గదిలో చేరడానికి మీరు ఆహ్వానించబడ్డారు",
```
Comment (dbkr) [inline src/i18n/strings/te.json]: Capitalisation of the I here unintentional?

File: src/i18n/strings/te.json [hunk 2]
```diff
@@ -147,24 +147,24 @@
...
     "and": "మరియు",
     "and one other...": "మరియు మరొకటి ...",
-    "%(names)s and one other are typing": "% (పేర్లు) లు మరియు మరొకటి టైప్ చేస్తున్నారు",
-    "%(names)s and %(count)s others are typing": "% (పేర్లు) లు మరియు% (లెక్క) లు ఇతరులు టైప్ చేస్తున్నారు",
+    "%(names)s and one other are typing": "%(names)s మరియు మరొకటి టైప్ చేస్తున్నారు",
+    "%(names)s and %(count)s others are typing": "%(names)s మరియు %(count)s ఇతరులు టైప్ చేస్తున్నారు",
     "An email has been sent to": "ఒక ఇమెయిల్ పంపబడింది",
     "A new password must be entered.": "కొత్త పాస్ వర్డ్ ను తప్పక నమోదు చేయాలి.",
-    "%(senderName)s answered the call.": "% (SenderName) s కు సమాధానం ఇచ్చారు.",
+    "%(senderName)s answered the call.": "%(SenderName)s కు సమాధానం ఇచ్చారు.",
```
Comment (dbkr) [inline src/i18n/strings/te.json]: capitalisation again

File: src/i18n/strings/te.json [hunk 3]
```diff
@@ -179,14 +179,14 @@
     "Can't connect to homeserver - please check your connectivity, ensure your <a>homeserver's SSL certificate</a> is trusted, and that a browser extension is not blocking requests.": "గృహనిర్వాహకులకు కనెక్ట్ చేయలేరు - దయచేసి మీ కనెక్టివిటీని తనిఖీ చేయండి, మీ <a> 1 హోమరుసు యొక్క ఎస్ఎస్ఎల్ సర్టిఫికేట్ </a> 2 ని విశ్వసనీయపరుచుకొని, బ్రౌజర్ పొడిగింపు అభ్యర్థనలను నిరోధించబడదని నిర్ధారించుకోండి.",
     "Can't load user settings": "వినియోగదారు సెట్టింగ్లను లోడ్ చేయలేరు",
     "Change Password": "పాస్వర్డ్ మార్చండి",
-    "%(senderName)s changed their profile picture.": "% (SenderName) వారి ప్రొఫైల్ చిత్రాన్ని మార్చారు.",
-    "%(senderDisplayName)s removed the room name.": "% (SenderDisplayName) s గది పేరు తొలగించబడింది.",
```
Comment (dbkr) [inline src/i18n/strings/te.json]: capitals

**Comment by lukebarnard1** [comment]
test failure suggests that the RTS client is alive and trying to get a team based on the team token... but it then doens't get a team object with a `name`.

**Pull request merged**